src/changelog/sources/markdown/filter.go

```diff
@@ -0,0 +1,38 @@
+package markdown
+
+import (
+	"io"
+	"reflect"
+
+	"github.com/gomarkdown/markdown"
+	"github.com/gomarkdown/markdown/ast"
+)
+
+// FilterRenderer wraps a markdown.Renderer skips calling it for certain node types.
+// This allows to use the default markdown renderer with types it does not support without it panicking.
+type FilterRenderer struct {
+	filtered map[reflect.Type]bool
+	markdown.Renderer
+}
+
+// NewFilterRenderer returns a FilterRenderer given an underlying renderer and a list of nodes whose type will be ignored.
+func NewFilterRenderer(inner markdown.Renderer, ignore ...interface{}) FilterRenderer {
```

Comment:
```suggestion
func NewFilterRenderer(inner markdown.Renderer, ignore ...ast.Node) FilterRenderer {
```

As it is supposed to be used with Node types, would it make sense to use the Node interface instead of any type?

Reply:
This is a nice idea, I like that.